Makefile

```diff
@@ -148,12 +150,22 @@ uninstall:
 	rm -f $(BINDIR)/ctr
 
 $(BUILD_DIR)/$(TARBALL): static-binaries vendor.conf
-	@BUILD_DIR=$(BUILD_DIR) TARBALL=$(TARBALL) VERSION=$(VERSION) ./hack/release.sh
+	@BUILD_DIR=$(BUILD_DIR) TARBALL=$(join $(TARBALL),-$(ARCH).tar.gz) VERSION=$(VERSION) ./hack/release.sh
 
-release: $(BUILD_DIR)/$(TARBALL)
+release: all-release
```

Comment:
we don't need `all-release` target here, we can use `$(addprefix sub-release-,$(ALL_ARCH))` here.